sklearn/linear_model/stochastic_gradient.py

```diff
@@ -1254,11 +1254,9 @@ class SGDRegressor(BaseSGDRegressor):
 
     learning_rate : string, optional
         The learning rate schedule:
-
         - 'constant': eta = eta0
-        - 'optimal': eta = 1.0 / (alpha * (t + t0)) [default]
-        - 'invscaling': eta = eta0 / pow(t, power_t)
-
```

Comment:
restore the blank line here (do not contain any space)

Reply:
Blank line restored. But again getting error :
W293 blank line contains whitespace
after running pep8 command. Also tried autopep8 but doesn't resolves.

Reply:
@bodhwani 
please make sure that your blank line do not contain white space, sometimes, after pressing enter, some while space will automatically be added to that black line.